internal/domain_errs/domain_errs.go

```diff
@@ -0,0 +1,7 @@
+package domainErrs
+
+import "errors"
+
+var ErrNotFound error = errors.New("record not found: ")
+var ErrBadData error = errors.New("incorrect data: ")
+var ErrInternal error = errors.New("internal server error: ")
```

Comment:
do we really need `: ` there? You have too many places where you're not placing any additional info
```suggestion
var ErrNotFound error = errors.New("record not found")
var ErrBadData error = errors.New("incorrect data")
var ErrInternal error = errors.New("internal server error")
```